src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/representation/CaseInteractionDetailsDTO.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.ons.ctp.integration.contactcentresvc.representation;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.LocalDateTime;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CaseInteractionDetailsDTO {
```

Comment:
I thought this was just plain old CaseInteractionDTO ? (without the Details bit)

Reply:
Renamed.